tools/c7n_gcp/c7n_gcp/query.py

```diff
@@ -244,6 +249,53 @@ def _fetch_resources(self, query):
     def augment(self, resources):
         return resources
 
+    @classmethod
```

Comment:
I was thinking class method when on resource type info, but if its on the manager, its probably fine to just let it be an instance method, which allows for using the project id already in ctx/settings versus sourcing it by one of its many env vars as done below.